src/profile-logic/import/chrome.js

```diff
@@ -118,6 +120,18 @@ type ProcessLabelsEvent = TracingEvent<{|
   args: { labels: string },
 |}>;
 
+type ProcessSortIndexEvent = TracingEvent<{|
```

Comment:
This is wild, they emit events for the sorting? Do you know how this works?

Optional: I'm curious in the review, so it might be nice to add a quick code comment.

Reply:
Well, the format is a list of events, so I'm not sure what the alternative would be - if you need information in the profile, you emit an event. I suppose you could have a "sorted list of threads" event will a full list, but it might be more complicated.
I haven't looked at the code that creates these events.